pom.xml

```diff
@@ -22,7 +22,7 @@
     </scm>
 
     <properties>
-        <otp.serialization.version.id>22</otp.serialization.version.id>
```

Comment:
I've increase to 23 in a recent PR. Can you bump it to 24?

Reply:
https://github.com/opentripplanner/OpenTripPlanner/commit/c6e6de804e35c04fa3cf0002beacbdb4368e1317